pkg/api/types.go

```diff
@@ -1160,6 +1161,14 @@ func (w *WindowsProfile) GetWindowsSku() string {
 	return KubernetesDefaultWindowsSku
 }
 
+// GetEnableWindowsUpdate gets the flag for enable windows update or returns the default value
+func (w *WindowsProfile) GetEnableWindowsUpdate() bool {
+	if w.EnableAutomaticUpdates != nil {
+		return *w.EnableAutomaticUpdates
+	}
+	return true
```

Comment:
Let's put a new const under `pkg/api/const.go`, e.g., `DefaultEnableAutomaticUpdates`. Set the value of that const to `true`. And then have this convenience method return that const. (instead of using the `true` bool literal here)